certified-connectors/Cognito Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1135 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Cognito Forms",
+    "description": "Cognito Forms lets you easily build powerful online forms, such as surveys, order forms, registration forms and more. For free.",
+    "x-ms-api-annotation": {
+        "status": "Production"
+    },
+    "contact": {
+      "name": "Cognito Forms Support",
+      "email": "support@cognitoforms.com",
+      "url": "https://www.cognitoforms.com/support"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.cognitoforms.com"
+    },
+    {
+      "propertyName": "Privacy Policy",
+      "propertyValue": "https://www.cognitoforms.com/legal/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity;Collaboration"
+    }
+  ],
+  "host": "www.cognitoforms.com",
```

Comment:
@sriyen-msft Yes, we would like to change it to `www.cognitoforms.com` **unless this will cause any issues for existing customers** (since we do have customers who have already connected via OAuth and are using our connector). As far as our product is concerned, these URIs are equivalent.